src/utils/searchHelpers.ts

```diff
@@ -200,3 +201,9 @@ export const syncParamsWithSearchFields = (params: URLSearchParams) => {
 
   return params;
 };
+
+export const deleteFromParameter = (params: URLSearchParams, navigate: NavigateFunction) => {
```

Comment:
Tolka det først som at man skulle fjerne noe vilkårlig _fra_ en parameter, ikke at man skulle fjerne faktiske _from_ param. Beskriver dette handlingen bedre? (Lov å være uenig her altså!)

```suggestion
export const resetPagination(Param?) = (params: URLSearchParams, navigate: NavigateFunction) => {
```

Reply:
Siden den gjør en navigate også kunne det vært `resetPaginationAndNavigate` eller noe i den duren også